tests/unit/src/services/submissionsService.spec.ts

```diff
@@ -1,128 +1,110 @@
+        id: "validId",
+        providerUserId: "validProviderUserId",
+        friendlyId: "validFriendlyId",
+        providerOfficeId: "validProviderOfficeId",
+        submissionTypeCode: "validSubmissionTypeCode",
+        submissionDate: "validSubmissionDate",
+        submissionPeriodStartDate: "validPeriodStart",
+        submissionPeriodEndDate: "validPeriodEnd",
+        scheduleId: "validScheduleId",
+        claims: undefined, // sitting as undefined -- not implemented yet
+      };
+
+      // Act - call transformer with the mock data
+      const result = transformSubmission(mockDataInput);
+
+      // Assert - check the data
 
       // type guarding - defensive coding
-    expect(isSubmission(result)).to.be.true;
+      expect(isSubmission(result)).to.be.true;
 
       // checks this is a valid object (Submission is interface - doesn't exist at runtime)
-    expect(result).to.be.an('object'); 
+      expect(result).to.be.an("object");
 
       // checks the data conforms to the right structure
-    expect(result).to.have.property('id', 'validId'); 
-    expect(result).to.have.property('friendlyId', 'validFriendlyId'); 
-    expect(result).to.have.property('providerUserId', 'validProviderUserId'); 
-    expect(result).to.have.property('providerOfficeId', 'validProviderOfficeId'); 
-    expect(result).to.have.property('submissionTypeCode', 'validSubmissionTypeCode'); 
-    expect(result).to.have.property('submissionDate', 'validSubmissionDate'); 
-    expect(result).to.have.property('submissionPeriodStartDate', 'validPeriodStart'); 
-    expect(result).to.have.property('submissionPeriodEndDate', 'validPeriodEnd'); 
-    expect(result).to.have.property('scheduleId', 'validScheduleId'); 
-
-    expect(result.claims).to.be.an('array');
-    })
+      expect(result).to.have.property("id", "validId");
+      expect(result).to.have.property("friendlyId", "validFriendlyId");
+      expect(result).to.have.property("providerUserId", "validProviderUserId");
+      expect(result).to.have.property("providerOfficeId", "validProviderOfficeId");
+      expect(result).to.have.property("submissionTypeCode", "validSubmissionTypeCode");
+      expect(result).to.have.property("submissionDate", "validSubmissionDate");
+      expect(result).to.have.property("submissionPeriodStartDate", "validPeriodStart");
+      expect(result).to.have.property("submissionPeriodEndDate", "validPeriodEnd");
+      expect(result).to.have.property("scheduleId", "validScheduleId");
+
+      expect(result.claims).to.be.an("array");
+    });
 
     it("should throw an error when the transformer is used on null", () => {
-    
-    // Arrange - set up the mock data
-    const mockDataInput = null;
-
-    // Act - call transformer with the mock data
-    // Assert - check it is thrown
-
-    expect(() => transformSubmission(mockDataInput)).to.throw('Invalid submissions item: expected object'); 
-    })
-  })
+      // Arrange - set up the mock data
```

Comment:
you don't need to add comments as the intent is clear in the name `mockDataInput`